client/src/featureform/sqlite_metadata.py

```diff
@@ -267,23 +267,65 @@ def createTables(self):
         VALUES ('provider', new.name, '', new.description, new.status);
         END;''')
 
+        # Tags and Properties tables
+        self.__conn.execute('''CREATE TABLE IF NOT EXISTS tags(
+          name text NOT NULL,
+          variant text,
+          type text,
+          tag_list text,
+          UNIQUE(name,variant));''')
```

Comment:
This should also include resource types? Imagine we have a source and a feature with the same name/variant

Reply:
Good catch! I missed this. I'll update the unique constraint.